composer.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "drupal/commerce_bluesnap",
+  "type": "drupal-module",
+  "description": "Provides Drupal Commerce integration with the BlueSnap Payment Gateway.",
+  "keywords": ["Drupal"],
+  "homepage": "https://www.drupal.org/project/commerce_bluesnap",
+  "license": "GPL-2.0+",
+  "authors": [
+    {
+      "name": "shabana.navas",
+      "homepage": "https://www.drupal.org/u/shabananavas",
+      "role": "Maintainer"
+    },
+    {
+      "name": "krystalcode",
+      "homepage": "https://www.drupal.org/u/krystalcode",
+      "role": "Co-maintainer"
+    }
+  ],
+  "support": {
+    "issues": "https://www.drupal.org/project/issues/commerce_bluesnap",
+    "source": "http://cgit.drupalcode.org/commerce_bluesnap"
+  },
+  "require": {
+    "drupal/commerce": "~2.0",
+    "shabananavas/php-bluesnap-sdk": "dev-develop"
+  }
```

Comment:
@shabananavas I assume that right now you're adding the repository directly in your application's `composer.json` file. We will need to publish through packagist, create an account at https://packagist.org/ and set it up to publish this repository.

Also, the requested branch should be the `master` branch here until we make a release for the SDK. If you need a specific branch while developing the Drupal module you can always override things at the application's `composer.json` file.